bin/cheribsdtest/cheribsdtest.c

```diff
@@ -373,6 +374,23 @@ cheribsdtest_run_test(const struct cheri_test *ctp)
 
 	(void)waitpid(childpid, &status, 0);
 
+	/**
+	 * If the test explicitly signalled failure for some reason, report
+	 * this first.
```

Comment:
```suggestion
	/*
	 * If the test explicitly signalled failure for some reason, report
	 * this first rather than reporting an expected failure that has
	 * not yet been triggered.
```